ajax/libs/anchor.js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "anchor.js",
+  "filename": "anchor.min.js",
+  "description": "This small but might really useful jQuery Plugin brings in an animation to all native section link and smoothly jumps at an element anywhere on your page.",
+  "keywords": [
+    "jquery-plugin",
+    "section",
+    "link",
+    "anchor",
+    "animation"
+  ],
+  "version": "1.0.6",
+  "author": "Cornel Boppart (https://github.com/cbopp-art)",
+  "license": "deprecated_GPL-2.0+",
```

Comment:
i knew that, but if there is any uncertain object, we should at least open an issue then send the PR.